sdk/storage/tests.yml

```diff
@@ -41,9 +42,7 @@ stages:
         - name: azure-storage-perf
           groupId: com.azure
       TestMode: 'LIVE'
-      TimeoutInMinutes: 60
-      Location: canadacentral
-      Clouds: 'Preview'
+      TimeoutInMinutes: 240
```

Comment:
If we're going beyond 60 minutes we should redesign the approach. It is also possible that you're running into many failures and retries which add up to large time. Normally live test suite for java completes below 30 minutes.

Reply:
This change has been rolled back